Artifacts/windows-deploymentgroup/InstallAndConfigureDeploymentGroupAgent.ps1

```diff
@@ -144,6 +144,76 @@ function Configure-DeploymentGroupAgent
 
     if ($runAsAutoLogon)
     {
+        # Create a PS session for the user to trigger the creation of the registry entries required for autologon
+        if ($windowsLogonAccount.Split("\").Count -eq 2)
+        {
+            $domain = $windowsLogonAccount.Split("\")[0]
+        }
+        else
+        {
+            $domain = Hostname
```

Comment:
Where does `Hostname` get defined? Are you expecting to use the literal string or is this supposed to be a variable you defined somewhere? Not sure I get its origin.

Reply:
Its an alias for hostname.exe, changed it to the use the env var $Env:ComputerName